artifacts/holesky/readme.md

```diff
@@ -2,3 +2,4 @@
 # Holesky
 
 Current deployed commit: b7ac032c7dd7511fa899f97a2561003c1941120f
+Last implementations upgrade commit: 691830e04ea838008d63cc5a76fe56a59550113a
```

Comment:
why a different line? just override the value above

Reply:
yeah, thought about this